tsconfig.json

```diff
@@ -1,10 +1,9 @@
 {
-  "//extends": "pick from: stylomatic/{node-library|react-library|nextjs|base}.json",
-  "extends": "stylomatic/node-library.json",
```

Comment:
Removed dependency of [stylomatic](https://github.com/jondot/stylomatic/tree/master); it has not been maintained.